### DISCUSSION TITLE
Update docs to clarify difference from MUI X Datagrid

### DISCUSSION THREAD

**Discussion by githorse** [KevinVandy/material-react-table#521] [Ideas]
The [feature comparison with MUI X](https://www.material-react-table.com/#feature-comparison) is very helpful. However, given how similar this grid looks to the MUI one, I think it would be helpful for the docs to include a paragraph explaining the motivation for this separate project. 

I'm currently evaluating both this grid and the MUI one for my company's needs and. Since 90% of the features are the same, it's not clear to me what the major advantages are of Material React Table. For instance, are there performance advantages to React Material Table? Different design philosophies? Do you expect the project to evolve in a different direction from MUI X?

**Comment by KevinVandy** [comment]
I've honestly never really used MUI X, mainly because some basic table features are hidden behind a paywall. Devs willing to pay MUI will enjoy a slightly more polished out of the box experience. MRT is 100% open source and free. It has a way smaller bundle size, but gets you 90% of the way there by default. MRT also tries to expose as much of its underlying API as possible for easier customization. 

I kind of think that my docs already pretty much say this, but feel free to PR anything to clear up confusion in the comparison chart.

**Comment by BigSamu** [comment]
When looking for "Material Table" in bing (https://www.bing.com/search?q=Material+Table) and scroll a bit downwards I get to this benchmark, but I don't know from where is coming from

<img width="500" alt="image" src="https://github.com/user-attachments/assets/a7cda1b7-cac2-4f69-9798-3208ad5c9d32">

<img width="550" alt="image" src="https://github.com/user-attachments/assets/cff860d1-4eaa-46a6-bc45-56e66ab48660">

That graph and table were created by the Material React Table team? I can't fin them in your website? At least here you can make a comparisson in terms of bundle size
